### ISSUE TITLE
Name and version

### ISSUE THREAD

**Issue by SzotsGabor** [joey-bednar/amelia#1]
Hello Joey,

The newest Amelia, allegedly version 0.11 displays Annie as its name. Also, no version is displayed. The file name is also simply Amelia, maybe Amelia0.11 would clarify which version it is.

BTW, I could find no Windows exe, I had to use my own compile but most people can't compile it for themselves.

Gabor Szots
CCRL testing group

**Comment by joey-bednar** [comment]
Hello,

Thank you for your comments and interest in my project.

- I agree that the UCI output should include the version number and just made that change.
- I will add the version to the names of the release binaries.
- I will upload Windows binaries very soon

I changed the name from "Annie" to "Amelia" to avoid conflict with another chess engine that was released a few months prior to mine. I changed the name in all places (including the release binary names) but did not go back in the git history to change this.

I should have made a minor release as soon as I made the name change because it is quite confusing. I will make a new release and `v0.12` will not have these issues going forward.
